src/components/Tooltip/Tooltip.jsx

```diff
@@ -54,8 +56,13 @@ export default class Tooltip extends React.PureComponent {
     return (
       <div
         style={style}
-        className={classnames(
-          `monday-style-tooltip monday-style-tooltip-${theme} padding-size-${paddingSize}`,
+        className={cx(
+          styles.mondayStyleTooltip,
+          "monday-style-tooltip",
+          styles[`${camelCase("monday-style-tooltip-" + theme)}`],
```

Comment:
why also converting "monday-style-tooltip-" to camel case?

Reply:
U mean we can do it like this or what?
```styles[`mondayStyleTooltip${camelCase(theme)}`]```